src/modules/fw_att_control/fw_att_control_params.c

```diff
@@ -500,7 +500,7 @@ PARAM_DEFINE_FLOAT(FW_MAN_P_MAX, 30.0f);
  * Flaps setting during take-off
  *
  * Sets a fraction of full flaps during take-off.
- * Also applies to flaperons if enabled in the mixer/allocation.
+ * Also applies to flaperons if enabled in control allocation.
```

Comment:
This feature (flaperons) are actually not yet enabled in CA - should we remove the comment for now? 
I'll currently cleaning up the interfaces between the controller, including dedicated flap, spoiler, landing gear etc modules, that work is needed to enable it. 
![image](https://user-images.githubusercontent.com/26798987/196633201-ab7fdd43-5c35-4011-9d65-05b02c9fb8a3.png)